network-api/networkapi/buyersguide/views.py

```diff
@@ -15,6 +15,7 @@
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
 
+from networkapi.wagtailpages.pagemodels.products import BuyersGuidePage
```

Comment:
not sure how to test my changes in this file

Reply:
We don't use this file anymore, it's part of the old PNI app, so we only need changes to the wagtailpages code

Reply:
Alright! Will revert these changes!